src/preprocessor.ts

```diff
@@ -19,11 +18,7 @@ export function process(
 ) {
   // transformOptions.instrument is a proxy for collectCoverage
   // https://github.com/kulshekhar/ts-jest/issues/201#issuecomment-300572902
-  const compilerOptions = getTSConfig(
-    jestConfig.globals,
-    jestConfig.rootDir,
-    transformOptions.instrument,
-  );
+  const compilerOptions = getTSConfig(jestConfig.globals, jestConfig.rootDir);
```

Comment:
`instrument` is used as a cache key. Unless the jestConfig passed in also contains a field (collectCoverage, maybe) that represents the same bit of data, it might be better to leave this in so that the cache is invalidated if this value changes.

Reply:
Ooh, you're thinking that while we might not use instrument, Jest might still, and thus it should invalidate our cache?

Reply:
yes, that's right

Reply:
I agree. I will add it back in.

Reply:
Wait, that doesn't make sense. We're caching our tsconfig here - we're still using the instrument in the getCacheKey, which should be the one jest cares about, yeah?

Reply:
you're right. It seems that instrument was removed from there as well. This comment should apply only to the part that Jest cares about - so it should be fine to remove that from the tsconfig part

Reply:
Yeah that makes sense.